test/e2e/flowvisibility_test.go

```diff
@@ -1145,9 +1263,7 @@ type ClickHouseFullRow struct {
 }
 
 func failOnError(err error, t *testing.T, data *TestData) {
-	if err != nil {
-		log.Errorf("%+v", err)
-		data.Cleanup(namespaces)
-		t.Fatalf("test failed: %v", err)
-	}
+	portForwardCmd.Process.Kill()
```

Comment:
Will this line cause an error if `failOnError` is called before the portForward process starts? For example, on Line 145 and 150.

Reply:
Good catch. I added a nil check for `portForwardCmd.Process` before killing it. Thanks.